mpf/platforms/fast/fast.py

```diff
@@ -244,8 +250,10 @@ def register_io_board(self, board):
 
     def _update_watchdog(self):
         """Send Watchdog command."""
-        if self.net_connection:
+        try:
             self.net_connection.send('WD:' + str(hex(self.config['watchdog']))[2:])
+        except:
```

Comment:
Why do we ignore all errors here? I am certain that this breaks the linter (which is not currently running in CI).